src/providers/anthropic/messages.ts

```diff
@@ -169,39 +170,64 @@ export class AnthropicMessagesProvider extends AnthropicGenericProvider {
     }
 
     try {
-      const response = await this.anthropic.messages.create(params, {
-        ...(typeof headers === 'object' && Object.keys(headers).length > 0 ? { headers } : {}),
-      });
-      logger.debug(`Anthropic Messages API response: ${JSON.stringify(response)}`);
+      if (shouldStream) {
+        // Handle streaming request
+        const stream = await this.anthropic.messages.stream(params, {
```

Comment:
Added streaming, since for `thinking: enabled: true`, I was seeing the following error from anthropic:

```API call error: Streaming is required for operations that may take longer than 10 minutes. See https://github.com/anthropics/anthropic-sdk-typescript#long-requests for more details```